packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -13,6 +14,12 @@ type NestedKeyOf<Obj extends object> = {
   [K in keyof Obj]: Obj[K] extends object ? NestedKeyOf<Obj[K]> : K;
 }[keyof Obj];
 
+export const ethereumNetworks = ['mainnet', 'goerli', 'kovan'];
+export const gnosisNetworks = ['gnosis', 'sokol', 'xdai'];
+export const polygonNetworks = ['polygon', 'mumbai'];
+
+export const deprecatedNetworks = ['kovan', 'xdai'];
```

Comment:
Can we maybe remove this and let the `getWeb3ConfigByNetwork` throw an error in case we want to get config for an unsupported network (i.e kovan (and I think soon Sokol as well))?